src/model/trust.page.model.ts

```diff
@@ -1,24 +1,19 @@
+import { BeneficialOwnerTypeChoice } from 'model/beneficial.owner.type.model';
+
 type TrustDetails = {
   id: string;
   name: string;
   createdDateDay: string;
   createdDateMonth: string;
   createdDateYear: string;
-  beneficialOwners: TrustBeneficialOwner[];
+  beneficialOwners: string[];
```

Comment:
Guessing, but given change in datatype, looks like this doesn't contain beneficial owners anymore. Are there BO ids in this string array now and should field name be renamed?

Reply:
changed to `beneficialOwnersIds`